kinder/ci/workflows/skew-kubelet-1.19-on-latest.yaml

```diff
@@ -9,6 +9,7 @@ vars:
   kubeletVersion: "{{ resolve `ci/latest-1.19` }}"
   kubernetesVersion: "{{ resolve `ci/latest` }}"
   ignorePreflightErrors: "KubeletVersion"
+  ginkgoSkip: "\\[MinimumKubeletVersion:(1.20|1.21)\\]"
```

Comment:
only this change is needed for:
https://github.com/kubernetes/kubernetes/issues/99909
but the rest are added so that the ginkgoSkip variable is properly updated with the introduction of new workflow YAMLs.